frontend/src/services/amendingLawsService.ts

```diff
@@ -29,3 +29,19 @@ export async function getAmendingLawXmlByEli(eli: string): Promise<string> {
     },
   })
 }
+
+/**
+ * Save the xml version of an amending law to the API.
+ *
+ * @param eli Eli of the amending law
+ * @param xml New xml of the amending law
+ */
+export async function putAmendingLawXml(eli: string, xml: string) {
+  return await apiFetch.raw<undefined>(`/amending-laws/${eli}`, {
```

Comment:
Todo: As discussed, I would recommend to use the regular call without `raw` and do any error handling in the `catch` block, see https://github.com/unjs/ofetch?tab=readme-ov-file#%EF%B8%8F-handling-errors